Sources/JExtractSwift/ImportedDecls.swift

```diff
@@ -44,18 +44,19 @@ public struct ImportedNominalType: ImportedDecl {
     TranslatedType(
       cCompatibleConvention: .direct,
       originalSwiftType: "\(raw: swiftTypeName)",
+      originalSwiftTypeKind: self.kind,
```

Comment:
TODO: we'll want to remove this and instead pass around a SwiftNominalTypeDeclaration instead, and base the name and kind off from that.